integrations/langchain/pyproject.toml

```diff
@@ -15,6 +15,14 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+databricks-dev = [
```

Comment:
where is this used?

Reply:
It's used in the integrations test PR in the private repo